### DISCUSSION TITLE
iOS抖音无水印快捷指令下载/iOS TikTok no watermark shortcut download

### DISCUSSION THREAD

**Discussion by Evil0ctal** [Evil0ctal/Douyin_TikTok_Download_API#104] [Announcements]
## iOS Shortcut/快捷指令

>  快捷指令只需手动安装一次，往后在运行时会自动连接API-V1进行更新检查。

> The shortcut command only needs to be manually installed once, and will automatically connect to API-V1 for update checking at runtime.

> 如果你愿意分享你魔改的快捷方式(捷径)，欢迎在下方留言，我会将你的快捷方式链接收录至此文档内，感谢你的工作(#^.^#)

> If you are willing to share the shortcut of your magic modification, please leave a message below, I will include your shortcut link in this document, thank you for your work (#^.^#)

[V7.0]
Release date: 2024/07/02
中文：
https://www.icloud.com/shortcuts/06f891a026df40cfa967a907feaea632
English：
https://www.icloud.com/shortcuts/06f891a026df40cfa967a907feaea632
Note:
重构快捷指令，使用TikHub.io的私有API保证稳定性，使用前需要先在快捷指令中填写自己的API Token。
Note:
Refactor shortcuts to use TikHub.io's private API to ensure stability. You need to fill in your own API Token in the shortcut before use.

[V6.0]
Release date: 2022/11/06
中文：
https://www.icloud.com/shortcuts/4465d514869e4ca585074d40328f3e0e
English：
https://www.icloud.com/shortcuts/58e3a2cbac784a6782f1031c6b1dd9f8
Note:
对最新的API-V1进行了适配，必须安装此更新才能正常使用捷径。
Note:
Adapted to the latest API-V1, this update must be installed to use shortcuts normally.

[V5.0]
Release date: 2022/07/18
中文/Chinese：
https://www.icloud.com/shortcuts/331073aca78345cf9ab4f73b6a457f97
英文/English：
https://www.icloud.com/shortcuts/83548306bc0c4f8ea563108f79c73f8d

[V4.0]
Release date: 2022/07/15
中文/Chinese：
https://www.icloud.com/shortcuts/25af5f6d9a9140e1a4e35c771313732f
英文/English：
https://www.icloud.com/shortcuts/0d37a661c1044ce4a428a84c13113c30

[V3.0]
Release date: 2022/04/16
中文/Chinese：
https://www.icloud.com/shortcuts/126820d2783748d1bdec95a223a02639

[V2.0]
Release date: 2022/04/06
中文/Chinese：
https://www.icloud.com/shortcuts/38df6ca6f54840e5af80b98bf52b9c3b

[V1.0]
Release date: 2022/02/06
中文/Chinese：
https://www.icloud.com/shortcuts/e8243369340548efa0d4c1888dd3c170

**Comment by stormblizzard** [comment]
大佬，v4.0更新后提示“无效URL”。 多个视频，中英文版本的shortcut都试了，都报这个error。
麻烦看一下是什么问题啊？
谢谢。

**Comment by Evil0ctal** [comment]
> 大佬，v4.0更新后提示“无效URL”。 多个视频，中英文版本的shortcut都试了，都报这个error。
> 
> 麻烦看一下是什么问题啊？
> 
> 谢谢。

你好 提供一下参数 视频链接

**Comment by stormblizzard** [comment]
https://vm.tiktok.com/ZMNHxogr2/
error是“”“未指定URL”。所有tiktok的都这样。抖音还没试过。
没设定别的参数，就是用ios的捷径正常下载的。大佬回复好快！
感谢！

**Comment by Evil0ctal** [comment]
> https://vm.tiktok.com/ZMNHxogr2/
> 
> error是“”“未指定URL”。所有tiktok的都这样。抖音还没试过。
> 
> 没设定别的参数，就是用ios的捷径正常下载的。大佬回复好快！
> 
> 感谢！

好的，我去检查一下。

**Comment by stormblizzard** [comment]
> > https://vm.tiktok.com/ZMNHxogr2/
> > error是“”“未指定URL”。所有tiktok的都这样。抖音还没试过。
> > 没设定别的参数，就是用ios的捷径正常下载的。大佬回复好快！
> > 感谢！
> 
> 好的，我去检查一下。

多谢多谢！

**Comment by Evil0ctal** [comment]
> > > https://vm.tiktok.com/ZMNHxogr2/
> 
> > > error是“”“未指定URL”。所有tiktok的都这样。抖音还没试过。
> 
> > > 没设定别的参数，就是用ios的捷径正常下载的。大佬回复好快！
> 
> > > 感谢！
> 
> > 
> 
> > 好的，我去检查一下。
> 
> 
> 
> 多谢多谢！

你好，刚刚测试了一下没有复现你的问题。

**Reply by Evil0ctal** [reply]
已经修复了

**Comment by stormblizzard** [comment]
> > > > https://vm.tiktok.com/ZMNHxogr2/
> > 
> > 
> > > > error是“”“未指定URL”。所有tiktok的都这样。抖音还没试过。
> > 
> > 
> > > > 没设定别的参数，就是用ios的捷径正常下载的。大佬回复好快！
> > 
> > 
> > > > 感谢！
> > 
> > 
> > > 
> > 
> > 
> > > 好的，我去检查一下。
> > 
> > 
> > 多谢多谢！
> 
> 你好，刚刚测试了一下没有复现你的问题。

额。。。。那我再试试。。。我这所有的tiktok视频都不行呢。。。重启手机试试。。。

**Comment by Evil0ctal** [comment]
> > > > > https://vm.tiktok.com/ZMNHxogr2/
> > > 
> > > 
> > > > > error是“”“未指定URL”。所有tiktok的都这样。抖音还没试过。
> > > 
> > > 
> > > > > 没设定别的参数，就是用ios的捷径正常下载的。大佬回复好快！
> > > 
> > > 
> > > > > 感谢！
> > > 
> > > 
> > > > 
> > > 
> > > 
> > > > 好的，我去检查一下。
> > > 
> > > 
> > > 多谢多谢！
> > 
> > 
> > 你好，刚刚测试了一下没有复现你的问题。
> 
> 额。。。。那我再试试。。。我这所有的tiktok视频都不行呢。。。重启手机试试。。。

问题找到了，可能是服务器尝试转换你获取的短链接时被TikTok服务器限制了，手动转换长链接之后可以成功解析，目前解决办法可能需要你尝试在本地部署一下API服务。
https://api.douyin.wtf/api?url=https://www.tiktok.com/@binjoy24/video/7112820250193792258

**Comment by stormblizzard** [comment]
好的。我试试。多谢！！
这个问题可能会被解决么？

**Comment by Evil0ctal** [comment]
> 好的。我试试。多谢！！ 这个问题可能会被解决么？

目前可能没有办法，你可以复制链接后在浏览器里打开，然后点击快捷指令将长链接粘贴在文本输入框中，或者你在自己的服务器上部署一下API服务，然后在快捷指令中修改API链接即可。

**Comment by Evil0ctal** [comment]
手动将链接
https://vm.tiktok.com/ZMNHxogr2/
转换为
[https://www.tiktok.com/@binjoy24/video/7112820250193792258](https://api.douyin.wtf/api?url=https://www.tiktok.com/@binjoy24/video/7112820250193792258)

**Comment by stormblizzard** [comment]
> > 好的。我试试。多谢！！ 这个问题可能会被解决么？
> 
> 目前可能没有办法，你可以复制链接后在浏览器里打开，然后点击快捷指令将长链接粘贴在文本输入框中，或者你在自己的服务器上部署一下API服务，然后在快捷指令中修改API链接即可。

> 

好的 明白了。。多谢！！

**Comment by geihub666** [comment]
6.0装了还是用不了

**Comment by geihub666** [comment]
更新了6.0，还是提示未指定url

**Comment by Git1015** [comment]
大佬修复了6.0了吗？跪求更新。盼回复

**Comment by Evil0ctal** [comment]
已经恢复了

**Comment by Git1015** [comment]
> 

还是不行，显示获取视频ID出错

**Reply by Git1015** [reply]
还是安装不了快捷指令，原因是 获取视频ID失败。我的抖音不是App store下载的，是在苹果自带浏览器下载的，跟这个有关系吗？

**Reply by Evil0ctal** [reply]
现在再试一下，跟下载源没有关系。

**Reply by Git1015** [reply]
> 现在再试一下，跟下载源没有关系。

![image](https://user-images.githubusercontent.com/117842786/201898425-93e4b658-6da8-432a-b756-3c2e98d71600.png)

**Reply by Evil0ctal** [reply]
你把视频的原始链接发给我看看。

**Reply by Git1015** [reply]
> 你把视频的原始链接发给我看看。

什么原始链接？这指令都没安装成功，我都没去tiktok，随便找个视频链接给你？

**Reply by Evil0ctal** [reply]
你的这张截图不是已经安装好快捷指令了吗？
根据你截图的错误信息，应该是获取视频ID的时候出错了，需要当时你运行快捷指令时的视频链接，这样我好在后端对这个问题进行修复。

如果你想问的是点击iCloud链接后无法安装快捷指令，请尝试更新至最新的iOS系统并使用代理排查网络原因。

**Reply by Git1015** [reply]
> 你的这张截图不是已经安装好快捷指令了吗？
> 根据你截图的错误信息，应该是获取视频ID的时候出错了，需要当时你运行快捷指令时的视频链接，这样我好在后端对这个问题进行修复。
> 
> 如果你想问的是点击iCloud链接后无法安装快捷指令，请尝试更新至最新的iOS系统并使用代理排查网络原因。

我发现长按视频就可以保存无水印的，这算成功了吗![image](https://user-images.githubusercontent.com/117842786/202086048-df2d4fd9-0ebe-4228-a599-421bb6731cc2.png)

**Reply by Git1015** [reply]
> 你的这张截图不是已经安装好快捷指令了吗？
> 根据你截图的错误信息，应该是获取视频ID的时候出错了，需要当时你运行快捷指令时的视频链接，这样我好在后端对这个问题进行修复。
> 
> 如果你想问的是点击iCloud链接后无法安装快捷指令，请尝试更新至最新的iOS系统并使用代理排查网络原因。

我发现长按视频就可以保存无水印的，这算成功了吗![image](https://user-images.githubusercontent.com/117842786/202086048-df2d4fd9-0ebe-4228-a599-421bb6731cc2.png)

**Reply by Evil0ctal** [reply]
复制分享链接，然后打开快捷指令，点击本项目的快捷指令，然后等待结果。

**Reply by Git1015** [reply]
> 复制分享链接，然后打开快捷指令，点击本项目的快捷指令，然后等待结果。

按照你说的可以正常下载了，我之前是这样下载![image](https://user-images.githubusercontent.com/117842786/202088937-01ee5586-c244-4af4-9fb3-a0b3597070a0.png)

**Reply by Evil0ctal** [reply]
点击捷径名称运行

**Reply by Git1015** [reply]
> 点击捷径名称运行

感谢大佬

**Comment by wujiang9100** [comment]
https://vm.tiktok.com/ZMFgg1uYs/
这个连接，用6.0中文的，下载的视频跟原视频对不上，用英文的就没问题哈

**Reply by Evil0ctal** [reply]
你好，我刚刚测试了一下，用英文的快捷指令，下载视频跟原视频也对不上，你用英文的快捷指令可以下载到正确的目标视频吗？

**Reply by wujiang9100** [reply]
是的，只有这个链接用英文快捷指令可以下载。 后续我又测试了其它的几个链接， 无论中文或者英文快捷指令下载的内容和原视频对不上

**Reply by Evil0ctal** [reply]
昨天已经修复了。

**Comment by tionxp** [comment]
![QQ截图20221211231413](https://user-images.githubusercontent.com/62242819/206912016-a5e81078-5705-40fa-83e7-ad40ad08a0d8.jpg)
抖音随便打开一个视频都显示这个，已经升级6.0了，无论是复制链接在快捷指令APP打开还是直接在抖音APP里分享点击快捷指令都提示这个

**Reply by Evil0ctal** [reply]
我刚刚测试了一下，没有复现这个问题，可以复制一个出错的视频链接给我吗？

**Reply by tionxp** [reply]
https://user-images.githubusercontent.com/62242819/207007834-b2049187-0588-4ce3-ba18-f526ba4b9c26.mp4

您好，随便找了个录了一下，您参考一下

**Reply by Evil0ctal** [reply]
是的，我的意思是，点击分享之后，然后你点击复制，把那个抖音的分享口令发到这里，我去在这边复现一下问题。

**Reply by tionxp** [reply]
9.76 rre:/ 复制打开抖音，看看【Yili多多的作品】没什么能难倒一个男人 # 青蛙嘴  https://v.douyin.com/hACub9Q/


又随便找了一个，我测试了下也是解析失败的，上面是分享链接

**Comment by TeslaLyon** [comment]
请问该捷径对 iOS版本有要求吗？我是 iOS14，浏览器访问 https://www.icloud.com/shortcuts/4465d514869e4ca585074d40328f3e0e，跳转到捷径 app，“执行” 下方一直显示 loading 图标。

**Reply by Evil0ctal** [reply]
对iOS版本是没有要求的，从你的描述中我只能猜测很有可能是你网络的问题，可以试试连接代理后再次尝试一下。

**Comment by Wanderxu88** [comment]
快捷指令用不了了

**Reply by Wanderxu88** [reply]
![image](https://user-images.githubusercontent.com/123281434/215538941-249654c8-1462-4d2b-961a-670edafd2a4e.png)

**Reply by Evil0ctal** [reply]
已经恢复了

**Reply by Wanderxu88** [reply]
好的

**Comment by dfinity-web3** [comment]
如果编辑快捷指令，更换成自己的api会不会速度更快些？

**Reply by Evil0ctal** [reply]
你可以自己部署一套API，然后把URL改成自己的域名，速度取决于你服务器的速度。

**Comment by maiiu-gf** [comment]
![image](https://user-images.githubusercontent.com/125494778/219111886-a3c762ec-0ea9-481d-891b-e7a81468436a.png)
抖音不能下载

**Reply by Evil0ctal** [reply]
再试试

**Comment by IDMlalala** [comment]
又出现这个问题了
![image](https://user-images.githubusercontent.com/94129571/221776813-ffb491e4-cf42-428d-b6d1-e9a25cb633ea.png)

**Reply by DDDD2000** [reply]
求大佬更新

**Reply by lzmmd** [reply]
求大佬更新

**Reply by Evil0ctal** [reply]
修复了

**Reply by lzmmd** [reply]
大佬我下午用的时候好使,但是我晚上好像又不行了

**Reply by cyay10** [reply]
求大佬更新

**Comment by xhnmt** [comment]
你好,快捷指令显示 请确保向获取url内容操作传递了有效url,是什么情况啊?

![IMG_5194-1](https://user-images.githubusercontent.com/45755242/228212778-27590244-0ae3-47e7-a7e8-15529844f57c.png)


下面是视频链接
3.53 reo:/ 复制打开抖音，看看【华商报的作品】海南省政府办公厅：民营企业家涉案人员 能不捕的不捕... https://v.douyin.com/Ahh1qKd/

**Reply by Evil0ctal** [reply]
再试试

**Reply by xhnmt** [reply]
可以了,3q

**Reply by helo1220** [reply]
厉害了老哥

**Comment by masami620** [comment]
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/129038750/22b41234-4dfb-49a5-acea-ef6074955ef6)
我不管是抖音还是tiktok,加载出来都这样，6.0版本已经是最新了，请问怎么解决？

**Comment by helo1220** [comment]
为啥抄送到我这里来了


发自我的iPhone

------------------ 原始邮件 ------------------
发件人: masami620 ***@***.***&gt;
发送时间: 2023年6月4日 01:01
收件人: Evil0ctal/Douyin_TikTok_Download_API ***@***.***&gt;
抄送: helo1220 ***@***.***&gt;, Comment ***@***.***&gt;
主题: 回复：[Evil0ctal/Douyin_TikTok_Download_API] iOS抖音无水印快捷指令下载/iOS TikTok no watermark shortcut download (Discussion #104)





 

 我不管是抖音还是tiktok,加载出来都这样，6.0版本已经是最新了，请问怎么解决？

**Comment by liuyanchen1015** [comment]
有出现“未指定URL”的错误了，大佬可能还需要再修复一下 😭 😭 😭

**Reply by Evil0ctal** [reply]
修复了

**Comment by kaipuzkp** [comment]
出现了“从输入中获取词典失败，因为快捷指令无法将文本转化为词典”的错误，试过只复制链接也不行，大佬能请教一下吗

**Reply by fyh080808** [reply]
请问解决了吗？

**Reply by kaipuzkp** [reply]
还是一样，没有改善

**Reply by Evil0ctal** [reply]
我晚点修复。

**Reply by realerikk0** [reply]
遇到了同样的问题，简单修改了一下快捷指令。本身的问题是请求api接口的时候没有将url编码导致类似“2.82 复制打开抖音，看看【杰叔叔捣鼓的图文作品】四十了 买了台能够打游戏的电脑却一个游戏。四十了 ... https://v.douyin.com/iY4GMGJX/ z@g.bN ULJ:/ 11/21 ”这种完全无法处理。加一个urlencode的小组件就好了。

顺便说一句，快捷指令编辑起来真nm难受。。。

**Comment by fyh080808** [comment]
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/138595740/11072e52-40e9-4b1d-b1ed-d5d9c919a3cd)
请问这个该如何解决？谢谢你！

**Reply by Evil0ctal** [reply]
我已经修复这个问题了

**Comment by anlo1220** [comment]
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/61088753/80e0f892-1362-427d-9109-c4ab4bc833b8)

**Reply by Evil0ctal** [reply]
修复了，你再试一下。

**Comment by ghost** [comment]
好用

**Comment by 2023djash** [comment]
大佬有没有 最新的，6.0 它提示更新

**Comment by 2023djash** [comment]


**Reply by Evil0ctal** [reply]
不客气😉

**Comment by JS063** [comment]
大佬，出现6.0版本更新这个问题好几天了，应该如何解决
![240ea9e7c2f52112b63be707a018a96](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/64151704/6fc710be-80eb-4dc3-9a22-743b2d3186fa)

**Reply by Evil0ctal** [reply]
晚点我解决。

**Comment by hww2732** [comment]
![Uploading IMG_0217.png…]()

**Reply by hww2732** [reply]
这种咋办

**Reply by hww2732** [reply]
显示快捷指令无法将文本转换为词典

**Reply by Evil0ctal** [reply]
服务器down了，我这边重新部署一下。

**Reply by Evil0ctal** [reply]
修复了

**Comment by JS063** [comment]
你好，v6.0的快捷指令无法下载，icloud找不到此快捷指令。
![774FA63B-C393-4682-978B-45B66A9DC4B3](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/64151704/b42e0c77-7752-4de6-b33c-95df1251be1e)

**Comment by KIRA-leebj** [comment]
Tiktok 可用
但是抖音不可以额：
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/141924558/432dedda-ee3a-47d4-a1de-876655ab6aee)

**Reply by wangdagen** [reply]
现在还是要什么更新，请问有办法吗？

**Comment by zhouc94** [comment]
服务器挂了，/api 状态码 500

**Reply by Evil0ctal** [reply]
修复了

**Reply by zhouc94** [reply]
https://api.douyin.wtf/ 服务器挂了

**Reply by Evil0ctal** [reply]
拉起来了

**Comment by anlo1220** [comment]
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/61088753/8d257b77-0b34-460f-ba7f-b4deef30cfcb)

**Reply by lzmmd** [reply]
大佬,这个好像也不太行呀,我也出现了这个问题,求大佬解决

**Reply by Evil0ctal** [reply]
各位稍安勿躁，最近生病了，等我几天哈

**Reply by lzmmd** [reply]
本来就是蹭大佬你的代码,我们有啥资格躁呀,早日康复

**Reply by anlo1220** [reply]
早日康复

**Reply by Evil0ctal** [reply]
感谢支持😊

**Reply by Evil0ctal** [reply]
已经修复了：）

**Reply by lzmmd** [reply]
牛啊大佬

**Reply by lzmmd** [reply]
求大佬再次出山，早上好像又不行了

**Comment by hiyuey3** [comment]
部分视频还是会显示快捷指令无法将文本转换为词典的错误

**Reply by Evil0ctal** [reply]
可以提供一些链接用于复现这个问题吗？

**Reply by hiyuey3** [reply]
测试发现：iOS最新抖音分享出来的大部分视频都报错，但是手动删除其他字符只保留链接解析正常。
例如：
5.15 q@r.RK vfb:/ 01/22 复制打开抖音，看看老娘与海🌊🌊🌊# 一起看海 # 去看可以治愈一切的... https://v.douyin.com/ieGWJYv2/
6.12 v@F.Hv yte:/ 03/01 复制打开抖音，看看能不能为你再跳一支舞# 白狐 # 翻唱  https://v.douyin.com/ieGWSVJ2/

不过我已经自行修改了快捷指令并解决了这个问题，以下是我的解决方案：
1. 在文本：\_INPUT\_ 下方增加一个匹配
2. 增加的匹配内容为：匹配 文本中的 [a-zA-z]+://[^\s]* 
3. 在 文本：/api?url=文本&minimal=true 把文本替换为 上述匹配问题解决。

**Reply by samuncleorange** [reply]
[a-zA-z]+://[^\s]*    这个匹配好像也不行    换成   https:\/\/[^\s]+   正常使用

**Comment by 663645265** [comment]
大佬，大佬快捷指令又不行了提示6.0版本需要检查更新

**Reply by samuncleorange** [reply]
https://www.icloud.com/shortcuts/8f8a3cb27831431b8dbea97d87732e26  作者修复之前可以拿这个应急  记得把api替换成自己的

**Comment by Flat6MT** [comment]
快捷指令又不行了提示6.0版本需要检查更新

**Reply by samuncleorange** [reply]
https://www.icloud.com/shortcuts/8f8a3cb27831431b8dbea97d87732e26  作者修复之前可以拿这个应急  记得把api替换成自己的

**Reply by wangdagen** [reply]
具体咋弄呢大佬

**Comment by PAULbigBA** [comment]
总是提醒需要更新到最新版本，请问怎么办呀

**Comment by darkmamu** [comment]
感谢！不过最近又坏了。

**Comment by helo1220** [comment]
有没有安卓手机用的方式呢



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年10月11日(周三) 晚上11:55
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [Evil0ctal/Douyin_TikTok_Download_API] iOS抖音无水印快捷指令下载/iOS TikTok no watermark shortcut download (Discussion #104)




 
感谢大佬，但是又不行了。

**Comment by openai1998** [comment]
抖音APP分享出来的短链接无法解析，Tik Tok正常解析，请教处理方法！[截图](https://img.xwyue.com/i/2023/12/27/658b981b67bdf.png)

**Comment by NPM400** [comment]
何时更新？

**Comment by yxx-z** [comment]
快捷指令有问题，https://api.douyin.wtf/ios  无法获取最新版本号。会一直校验版本。去掉这个后，快捷指令无法将文本转换为词典。

求修复

**Reply by Evil0ctal** [reply]
嗯嗯 我晚点适配一下

**Reply by Htin** [reply]
还不能用呀

**Comment by pcmzhn** [comment]
求更新！！！

**Comment by Evil0ctal** [comment]
各位，我准备基于TikHub.io的私有API写一个新的快捷指令，因为旧的快捷指令依赖于Demo站点的接口，但是由于风控越来越严重，Demo站点已经无法保证可用性了。

**Comment by Evil0ctal** [comment]
[V7.0]
Release date: 2024/07/02
中文：
[icloud.com/shortcuts/06f891a026df40cfa967a907feaea632](https://www.icloud.com/shortcuts/06f891a026df40cfa967a907feaea632)
English：
[icloud.com/shortcuts/06f891a026df40cfa967a907feaea632](https://www.icloud.com/shortcuts/06f891a026df40cfa967a907feaea632)
Note:
重构快捷指令，使用TikHub.io的私有API保证稳定性，使用前需要先在快捷指令中填写自己的API Token。
Note:
Refactor shortcuts to use TikHub.io's private API to ensure stability. You need to fill in your own API Token in the shortcut before use.

**Comment by Limbo199608** [comment]
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/159458055/73576221-d621-4c20-81d9-9dbde08a1d2f)![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/159458055/d23e54b3-e54a-4ae7-b869-b36c99602407)这是什么情况？

**Reply by aio521** [reply]
这个解决了吗怎么弄的

**Reply by Evil0ctal** [reply]
我在iOS18.1环境中测试没有发现问题

**Reply by cat-walk** [reply]
> 我在iOS18.1环境中测试没有发现问题

大佬，是偶现的，你可以换个视频试试，有差不多30%的概率出现。

**Reply by Evil0ctal** [reply]
> > 我在iOS18.1环境中测试没有发现问题
> 
> 大佬，是偶现的，你可以换个视频试试，有差不多30%的概率出现。

了解，你看看能不能发几个视频我测试一下

**Reply by 404area** [reply]
9.20 复制打开抖音，看看【阿美呀🎤的作品】# 无声卡清唱 # 翻唱 # 相思 # 毛阿敏 #... https://v.douyin.com/CeiW7oxG/ a@N.Wz wSY:/ 08/24 

我也有这个问题，自己没测试成功过

**Reply by Evil0ctal** [reply]
> 9.20 复制打开抖音，看看【阿美呀🎤的作品】# 无声卡清唱 # 翻唱 # 相思 # 毛阿敏 #... https://v.douyin.com/CeiW7oxG/ a@N.Wz wSY:/ 08/24 
> 
> 
> 
> 我也有这个问题，自己没测试成功过

好的，我用这个链接进行测试一下。

**Reply by cat-walk** [reply]
> > > 我在iOS18.1环境中测试没有发现问题
> > 
> > 大佬，是偶现的，你可以换个视频试试，有差不多30%的概率出现。
> 
> 了解，你看看能不能发几个视频我测试一下

0.51 复制打开抖音，看看【假装画画的小贝的作品】其实我也有点社恐# 内向社恐  https://v.douyin.com/CeiJAhfGo/ WMj:/ 12/02 L@w.fB

**Reply by Evil0ctal** [reply]
![image](https://github.com/user-attachments/assets/9b5cf8ca-5460-470f-bfa4-9da500b46077)

我用这个分享文本测试了一下，好像也没有出现失败的情况。

**Comment by Hushsuh** [comment]
您好，請問“使用前需要先在快捷指令中填写自己的API Token”是什麼意思，具體應該怎麼填寫api token?

**Reply by Evil0ctal** [reply]
需要在TikHub.io注册 然后签到获取余额 然后编辑快捷指令 填写你的API Token

**Comment by XWFeng511** [comment]
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/129677747/78889ea8-092c-4f74-898d-3206ce1de479)
ios18用不了

**Reply by Evil0ctal** [reply]
使用方式不对

**Reply by XWFeng511** [reply]
> 对

![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/129677747/64ba9b54-2bf9-4215-a377-5a0dcaae1b34)这个一直不行，这URL是什么

**Reply by XWFeng511** [reply]
求解答

**Reply by Evil0ctal** [reply]
复制下面这段话然后运行快捷指令：

5.84 复制打开抖音，看看【迟遇的图文作品】倘若遇到那个人 那就永远不要分开。# 文案 # 日... https://v.douyin.com/i6mu45GD/ H@i.ca fBt:/ 04/08

**Reply by XWFeng511** [reply]
> 复制下面这段话然后运行快捷指令：
> 
> 5.84 复制打开抖音，看看【迟遇的图文作品】倘若遇到那个人 那就永远不要分开。# 文案 # 日... https://v.douyin.com/i6mu45GD/ H@i.ca fBt:/ 04/08 
复制运行了还是像图一那样子

**Reply by XWFeng511** [reply]
> 复制下面这段话然后运行快捷指令：
> 
> 5.84 复制打开抖音，看看【迟遇的图文作品】倘若遇到那个人 那就永远不要分开。# 文案 # 日... https://v.douyin.com/i6mu45GD/ H@i.ca fBt:/ 04/08 
Emo了

**Reply by XWFeng511** [reply]
就不应该升这个ios18

**Reply by Evil0ctal** [reply]
你快捷指令里填写自己的API Key了吗

**Reply by XWFeng511** [reply]
> 你快捷指令里填写自己的API Key了吗

想问一下下怎么知道自己的Api key是什么🫣

**Reply by Evil0ctal** [reply]
晚点我录一个视频教程吧

**Reply by XWFeng511** [reply]
> 晚点我录一个视频教程吧

好的

**Reply by XWFeng511** [reply]
![image](https://github.com/Evil0ctal/Douyin_TikTok_Download_API/assets/129677747/c0defe13-1a1b-48a2-8073-f89031cded82)

这个API key 是付费的吗 签到也签不了那么多的额度

**Reply by Evil0ctal** [reply]
对的

**Comment by aio521** [comment]
未指定URL，下载失败
![71ca0a4f97d1c51f205f7c8cab575305](https://github.com/user-attachments/assets/c4e3f182-31e0-4ba8-b52e-b014662806dd)
![731921614fc50bdc1750d2a93d47a674](https://github.com/user-attachments/assets/b61484b6-6d85-4981-b1dc-c0d0d4a6177d)

**Comment by birdyabc** [comment]
大佬 ios18 填了Api key 还是显示未指定URL，该怎么解决啊

**Comment by Lorryrorschach** [comment]
实效， url 无效

**Comment by yunsxk** [comment]
提示需要正确的URL指向

**Comment by jacob007-art** [comment]
大佬 这个还在维护吗？ 求更新

**Comment by zoffypal** [comment]
根据讨论测试修复了一下，目前可以使用了.

[抖音TikTok无水印下载 V7.0.zip](https://github.com/user-attachments/files/31536576/TikTok.V7.0.zip)

在"\___INPUT\___"后面加上一个 https url 的正则匹配.
然后服务器地址这里，删除url 中"v1/"这部分，base64编码的这部分也删除，我测试的base64编码无法正确识别. 如果是自己部署的服务器就改成对应的地址.

<img width="1170" height="1977" alt="image" src="https://github.com/user-attachments/assets/92988847-3d8c-4146-8b6d-497b3b7c6fb0" />
<img width="1170" height="2373" alt="image" src="https://github.com/user-attachments/assets/9c87378e-6462-41c0-96bc-4a29a7da9f41" />

顺便吐槽下快捷方式，编辑起来真是难受. 更改了就保存，没有撤销，插入一个动作还得从最后面慢慢往前拖动
